solver/solver.go

```diff
@@ -26,30 +26,35 @@ import (
 )
 
 type LLBOpt struct {
-	SourceManager    *source.Manager
-	CacheManager     cache.Manager
-	Worker           worker.Worker
-	InstructionCache InstructionCache
-	ImageSource      source.Source
+	WorkerController *worker.Controller
```

Comment:
follow-up: I guess this isn't `LLBOpt` anymore as the controller is not LLB-specific. Only the workers it manages are.